scripts/scil_compute_asym_odf_metrics.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python3
+"""
+Script to compute various metrics derivated from asymmetric ODF.
+
+These metrics include an asymmetric peak directions image , a number of peaks
+(nupeaks) map, a cosine-similarity-based asymmetry map [1] and an odd-power map
+defined as the ratio of the L2-norm of odd SH coefficients on the L2-norm of
+all SH coefficients.
```

Comment:
Can you cite your ISMRM abstract maybe?  Should you not? 

Maybe it would help the user to have a little 1 sentence summary of the metric and its range, e.g.
nupeaks map is the equivalence of nufo for assymetric ODFs with typical range as integers between [0, ..., N]

Reply:
Added reference and description of metrics in b009b53